File: 02_EstructurasControl/Repetitivas/For/CSharp/Numero_de_combinaciones.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Numero_de_combinaciones
{
    internal class Program
    {
        static void Main(string[] args)
        {

            //Calcula el numero de combinaciones que se puede realizar con nª de polos y pantalones

            Console.WriteLine("Ingresa el numero de polos: ");
            int polos = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Ingresa el numero de pantalones: ");
            int pantalones = Convert.ToInt32(Console.ReadLine());
            int combinaciones = 0;

            for (int i = 1; i <= polos; i++)
            {
                for (int j = 1; j <= pantalones; j++)
                {
                    combinaciones++;
                    
                }
                

            }
            Console.WriteLine("Se puede realizar: " + combinaciones + " combinaciones");
            Console.ReadLine();
        }
    }
}

File: 02_EstructurasControl/Repetitivas/For/Python/Numero_de_combinaciones.py
# Calcula el número de combinaciones que se puede realizar con n° de polos y pantalones

polos = int(input("Ingresa el número de polos: "))
pantalones = int(input("Ingresa el número de pantalones: "))

combinaciones = 0

for i in range(1, polos + 1):        
    for j in range(1, pantalones + 1):   
        combinaciones += 1

print("Se puede realizar:", combinaciones, "combinaciones")
